src/accessibility/outputs.js

```diff
@@ -8,58 +8,67 @@
+ * Creates a screen reader-accessible description for shapes on the canvas.
+ * `textOutput()` adds a general description, list of shapes, and
+ * table of shapes to the web page.
  *
- * <code class="language-javascript">textOutput()</code> and <code class="language-javascript">textOutput(FALLBACK)</code>
- * make the output available in <a href="https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Hit_regions_and_accessibility" target="_blank">
- * a sub DOM inside the canvas element</a> which is accessible to screen readers.
- * <code class="language-javascript">textOutput(LABEL)</code> creates an
- * additional div with the output adjacent to the canvas, this is useful
- * for non-screen reader users that might want to display the output outside
- * of the canvas' sub DOM as they code. However, using LABEL will create
- * unnecessary redundancy for screen reader users. We recommend using LABEL
- * only as part of the development process of a sketch and removing it before
- * publishing or sharing with screen reader users.
+ * The general description includes the canvas size, canvas color, and number
+ * of shapes. For example,
+ * `Your output is a, 100 by 100 pixels, gray canvas containing the following 2 shapes:`.
+ *
+ * A list of shapes follows the general description. The list describes the
+ * color, location, and area of each shape. For example,
+ * `a red circle at middle covering 3% of the canvas`. Each shape can be
+ * selected to get more details.
+ *
+ * `textOutput()` uses its table of shapes as a list. The table describes the
+ * shape, color, location, coordinates and area. For example,
+ * `red circle location = middle area = 3%`. This is different from
+ * <a href="#/p5/gridOutput">gridOutput()</a>, which uses its table as a grid.
+ *
+ * The `display` parameter is optional. It determines how the description is
+ * displayed. If `LABEL` is passed, as in `textOutput(LABEL)`, the description
+ * will be visible in a div element next to the canvas. Using `LABEL` creates
+ * unhelpful duplicates for screen readers. Only use `LABEL` during
+ * development. If `FALLBACK` is passed, as in `textOutput(FALLBACK)`, the
+ * description will only be visible to screen readers. This is the default
+ * mode.
+ *
+ * Read
+ * <a href="/learn/labeling-canvases.html">How to label your p5.js code</a> to
+ * learn more about making sketches accessible.
  *
  * @method textOutput
- * @param  {Constant} [display] either FALLBACK or LABEL
+ * @param  {Constant} [display] either FALLBACK or LABEL.
  *
  * @example
  * <div>
  * <code>
- * textOutput();
- * background(148, 196, 0);
- * fill(255, 0, 0);
- * ellipse(20, 20, 20, 20);
- * fill(0, 0, 255);
- * rect(50, 50, 50, 50);
+ * function setup() {
+ *   textOutput();
+ *   background(200);
+ *   fill(255, 0, 0);
+ *   circle(20, 20, 20);
+ *   fill(0, 0, 255);
+ *   square(50, 50, 50);
+ *
+ *   describe('A red circle and a blue square on a gray background.');
```

Comment:
Would it make sense to explain the use case of using both textOutput() and describe()?